Service-level agreements (#2452)
SoftwareAG/c8y-docs: content/concepts/platform-sla.md

---
title: Service-level agreement
layout: bundle
sector:
  - getting_started
weight: 90
---

> <h3 style="margin-top: -1em; padding-top: 0">for {{< product-c8y-iot >}} platforms</h3>

Reviewer: To make it very clear from the beginning, that this document does not refer to any extension, I would change the title to something like "base platform"

**This Agreement** is made between {{< company-c8y >}} ("Provider") and the Customer ("Customer") who utilizes {{< product-c8y-iot >}} Platform ("Service") for managing Internet of Things ("IoT") devices ("IoT devices", "devices") on Provider's cloud instances ("software-as-a-service", "SaaS").

### Service description

> {{< product-c8y-iot >}} is a comprehensive Internet of Things (IoT) platform designed to enable seamless connectivity, management, and control of IoT devices. The platform provides a range of capabilities including:

Reviewer: I suggest to explicitly explain what is in scope for this service level agreement and what is out of scope

`This service level agreements covers all the Cumulocity IoT base platform services which includes:`

Author: This is the purpose of the service description (i.e., applies only to SaaS, not to the other editions). For the components, there are two possible approaches:

a) I exclude everything outside of the base platform (LWM2M, CDH, ...), which means that we have no service-level for these services. That would be a new situation and potentially not accepted by customers.
b) I leave the slightly vague "which may require separate agreements", which means that this document applies unless there are additional terms indexed there. That would be the current situation/customer expectation.

I would stick with b). That would be also a bit closer to the discussion that we had during lunch (of having one base/umbrella document). Ok?

Author: P.S. Reading a bit further I noticed that it would mean also to remove the part about "excluding optional features" from the scope description at the bottom of the paragraph.

Author: I have simplified the  "Service description" part stating only that this applies to the "base platform".

* **Device connectivity and management:** The platform facilitates the connection, data collection, and remote control of IoT devices. These functionalities can be accessed through the {{< product-c8y-iot >}} Device Management application or via application programming interfaces (APIs; REST/MQTT).
* **Data visualization:** Users can visualize their connected devices and corresponding data within the {{< product-c8y-iot >}} Cockpit, offering an intuitive interface for monitoring and managing IoT deployments.
* **Real-time and batch data processing:** {{< product-c8y-iot >}} supports both real-time and batch processing of IoT data, enabling users to efficiently manage and analyze large volumes of data as it is generated.

> * **Extended platform capabilities:** The platform's functionality can be further enhanced with additional features, which may require separate agreements:

Reviewer: I suggest to make it very clear to state what is not in scope of this Service Level Agreement.

`The platform's functionality can be further enhanced with additional features. These extended platform capabilities that are not in-scope of this Service Level Agreement.
Capabilities that are out-of-scope are:`

Author: See comment above.

    * **Additional connectivity options:** This includes extended connectivity support such as LPWAN and LWM2M.

>     * **Enhanced applications:** Users can leverage value-added Cumulocity applications from the Cumulocity product catalog, e.g., for advanced analytics and digital twin management.

Reviewer: This is too generic. The Digital Twin Manager is part of our standard product offering. We should explicitly exclude the DataHub product.

Author: See comment above.

>     * **Enhanced applications:** Users can leverage value-added Cumulocity applications from the Cumulocity product catalog, e.g., for advanced analytics and digital twin management.

Reviewer 2: ```suggestion
    * **Enhanced applications:** Users can leverage value-added {{< product-c8y-iot >}} applications from the {{< product-c8y-iot >}} product catalog, for example, for advanced analytics and digital twin management.
```

>     * **Custom Application Development:** The platform supports the creation of custom applications using {{< product-c8y-iot >}} Microservices and {{< product-c8y-iot >}} Web SDK.

Reviewer 3: Perhaps an idea to add information to make customers who utilizes Cumulocity IoT Microservices aware that there is a dedicated SLA definition for Microservices hosting on the Cumulocity IoT Microservices platform. 

- https://cumulocity.com/docs/microservice-sdk/microservices-sla/

Author: Added.

>     * **Custom application development:** The platform supports the creation of custom applications using {{< product-c8y-iot >}} Microservices and {{< product-c8y-iot >}} Web SDK. Services for Microservice Hosting are outlined in a [separate service-level agreement](/microservice-sdk/microservices-sla/).

Reviewer: We should also mention the WebSDK and UI Extensions here. In particular, we do also provide the ability to install community extensions easily. These community extensions are not part of the product and are not covered with this SLA.

Author: I added a sentence that the custom applications are not in scope of the SLA.

>     * **Custom application development:** The platform supports the creation of custom applications using {{< product-c8y-iot >}} Microservices and {{< product-c8y-iot >}} Web SDK. Services for Microservice Hosting are outlined in a [separate service-level agreement](/microservice-sdk/microservices-sla/).

Reviewer 2: ```suggestion
    * **Custom application development:** The platform supports the creation of custom applications using {{< product-c8y-iot >}} microservices and the {{< product-c8y-iot >}} Web SDK. Services for Microservice Hosting are outlined in a [separate service-level agreement](/microservice-sdk/microservices-sla/).
```

>     * **Enterprise-grade services:** Users can provide derived services to their own customers, utilizing Enterprise Functions and Multi-Tenancy features.

Reviewer 2: ```suggestion
    * **Enterprise-grade services:** Users can provide derived services to their own customers, utilizing enterprise functions and multi-tenancy features.
```

> This service-level agreement applies solely to {{< product-c8y-iot >}} Basic, {{< product-c8y-iot >}} Advanced and {{< product-c8y-iot >}} Enterprise Dedicated operated by {{< company-c8y >}}, excluding optional features. {{< product-c8y-iot >}} Software-Operated-as-a-Service and {{< product-c8y-iot >}} Edge are outside the scope of this agreement.

Reviewer: In the future, we are not referring to Basic, Advanced and Enterprise Dedicated anymore. Rather I would state that this Service Level Agreement only covers SaaS and excludes On-Prem, SoaaS and Edge.

Author: Changed the sentence accordingly.

> This service-level agreement applies solely to {{< product-c8y-iot >}} Basic, {{< product-c8y-iot >}} Advanced and {{< product-c8y-iot >}} Enterprise Dedicated operated by {{< company-c8y >}}, excluding optional features. {{< product-c8y-iot >}} Software-Operated-as-a-Service and {{< product-c8y-iot >}} Edge are outside the scope of this agreement.

Reviewer 2: @eickler @nneuerburg If we  introduce these "marketing" terms in the user documentation (until now we tried to avoid that), shouldn´t we at some place describe the difference or relationship between Basic, Advanced, Enterprise Dedicated on the one hand and Standard, Enterprise and Management tenants on the other side? Using "Enterprise" in two different contexts can cause confusion (and did in the past). If we introduce these terms here we should add them to the glossary and explain, shouldn´t we?

Author: I have removed these. 

Regarding the terms: These are the names for our services as purchased by the customer. So they should be sufficiently explained on the cumulocity.com web site ... eventually.

### Service features

The {{< product-c8y-iot >}} platform offers a comprehensive set of features designed to support the needs of enterprise IoT deployments, ensuring robust performance, security, and flexibility. Key service features include:

* **Real-time data processing and storage:** The platform supports real-time data processing, enabling instant insights and actions based on IoT data. Data storage is customer-configurable, allowing you to define retention periods according to your specific requirements.

> * **Enterprise-grade user and permission management:** {{< product-c8y-iot >}} includes advanced user and permission management features, allowing organizations to maintain strict control over access and operations within the platform.

Reviewer: I would not go into functionality here (Real-Time data processing, permission management) as this would duplicate our documentation in other places and we have made clear in the previous section what is in / out of scope.

Author: Right, that is redundant and not very structured. I'll cut it down and restructure it.

* **Proactive monitoring and management:**
    * {{< product-c8y-iot >}} includes 24x7 platform monitoring, with a publicly accessible status page and real-time status notifications to keep you informed.
    * The platform’s availability, capacity, and performance are actively managed to ensure consistent and reliable operation.
* **Information security compliance:**
    * We operate the platform in compliance with SOC II and ISO 27001 standards, encompassing a wide range of security measures, including vulnerability management, security incident management, DDoS protection, intrusion detection, and encryption of data both in transit and at rest.
* **Business continuity and resilience:**
    * {{< product-c8y-iot >}} adheres to ISO 22301 standards for business continuity management (BCM), ensuring resilience against zone outages with zone redundancy.
    * Regular backups are maintained with a 30-day retention period, a Recovery Time Objective (RTO) of 12 hours, and a Recovery Point Objective (RPO) of 24 hours.
    * Cumulocity conducts regular drills to validate its disaster recovery procedures.

> * **API compatibility management:** The platform maintains compatibility of APIs as outlined in its[ Compatibility Policy](/concepts/compatibility-policy/). Transport protocols are managed in accordance with this policy, ensuring consistent and reliable API interactions.

Reviewer 2: ```suggestion
* **API compatibility management:** The platform maintains compatibility of APIs as outlined in its[ Compatibility policy](/concepts/compatibility-policy/). Transport protocols are managed in accordance with this policy, ensuring consistent and reliable API interactions.
```

* **Data ownership and portability:** As the data processor, {{< product-c8y-iot >}} ensures that customers retain full ownership of their data. Customers can export their data at any time using the provided APIs, ensuring control and flexibility over their information.

> * **Continuous maintenance and upgrades:** The platform undergoes regular maintenance and upgrades to ensure optimal performance and security. These upgrades happen transparently and without involving Customers.

Reviewer 2: "Customer" is sometimes capitalized (which is obviously the case as it is introduced as proper name at the top) and sometimes it is not as in the following sentences. Does that follow any rule?

Author: Yeah, the rule is that if I wrote it, it's lower case, and if a legal dude wrote it, it's upper case ;-) I changed it now to be consistently upper case.

For some customers with legacy service agreements, these features may differ in accordance with the terms of their original agreements.

### Customer responsibilities

As a valued customer of the {{< product-c8y-iot >}} platform, we request your acknowledgment of the following responsibilities to ensure the continued security, performance, and efficiency of the service:

> ##### **Security Management**

Reviewer 2: ```suggestion
##### **Security management**
```

* **Proactive monitoring and management:**
    * {{< product-c8y-iot >}} includes 24x7 platform monitoring, with a publicly accessible status page and real-time status notifications to keep you informed.
    * The platform’s availability, capacity, and performance are actively managed to ensure consistent and reliable operation.
* **Information security compliance:**
    * We operate the platform in compliance with SOC II and ISO 27001 standards, encompassing a wide range of security measures, including vulnerability management, security incident management, DDoS protection, intrusion detection, and encryption of data both in transit and at rest.
* **Business continuity and resilience:**
    * {{< product-c8y-iot >}} adheres to ISO 22301 standards for business continuity management (BCM), ensuring resilience against zone outages with zone redundancy.
    * Regular backups are maintained with a 30-day retention period, a Recovery Time Objective (RTO) of 12 hours, and a Recovery Point Objective (RPO) of 24 hours.

>     * Cumulocity conducts regular drills to validate its disaster recovery procedures.

Reviewer 2: ```suggestion
    * {{< company-c8y >}} conducts regular drills to validate its disaster recovery procedures.
```

Reviewer 2: Please use {{< company-c8y >}} when referring to the company "Cumulocity"

### Cloud service credit commitment

#### Credit calculation

If the service is available for less than the availability outlined above during any full calendar month during the cloud services term, Customer will be eligible for a service credit for the particular service in accordance with the formula below (a “Service Credit”).

| Monthly availability | Percentage of the pro-rata monthly Cloud Service Fee for the covered cloud service |
| -------------------- | ---------------------------------------------------------------------------------- |
| 99.50% to < 99.90%   | 10%                                                                                |
| 99.50% to < 99.00%   | 15%                                                                                |
| < 99.00%             | 25%                                                                                |

> #### Credit Request

Reviewer 2: ```suggestion
#### Credit request
```

Customer must submit all requests for Service Credits by filing a request ("Service Credit Request") to support, including the necessary information to evaluate the request, including:

> 1. the date, time and duration of the incident giving rise to the Service Credit request (the “Incident”);

Reviewer 2: We usually use sentence capitalization in list items

I am a bit confused about all the capitalized terms. Is "Incident" also a fixed term which has been introduced somewhere? And Credit?

Author: Fixed.

2. a detailed description of the Incident, including any measures taken by Customer to resolve
the issue;
3. the tenant, number of Customer users and location(s) of Customer users affected by the incident (if applicable); and
4. any additional information reasonably requested by Supplier necessary to validate the Incident.

Supplier must receive the Service Credit Request within fourteen (14) days from the occurrence of the Incident. Supplier will evaluate the Service Credit Request as soon as all information necessary to review the Service Credit Request is received. Supplier will use commercially reasonable efforts to process complete Service Credit Requests during the subsequent calendar month and within thirty (30) days of receipt. If the Incident is confirmed by Supplier and gives rise to a Service Credit, Supplier shall provide Customer with a refund within thirty (30) days of Suppliers determination. The total amount credited to customer in a particular year under this SLA shall not exceed 5% of the annual fee (exclusive of any taxes) paid by the Customer for the affected services.

> #### Requirements and Exceptions

Reviewer 2: ```suggestion
#### Requirements and exceptions
```

> Customer must be current on any payment obligations owed to Supplier and in compliance with the terms of the Agreement and the Order Form in order to be eligible to receive Service Credits. The Service Availability commitments do not apply to any performance or availability issues:

Reviewer 2: ```suggestion
Customer must be current on any payment obligations owed to Supplier and in compliance with the terms of the Agreement and the order form in order to be eligible to receive Service credits. The Service availability commitments do not apply to any performance or availability issues:
```

1. Due to acts or conditions outside of Supplier’s reasonable control, including, but not limited
to, a Force Majeure event as defined in the agreement above;

> 2. Initiated by Supplier to protect the Cloud Services or Customer Data from unauthorized

Reviewer 2: ```suggestion
2. Initiated by Supplier to protect the Cloud Services or Customer data from unauthorized
```

access or loss;
3. Caused by Customer’s use of services, hardware, or software not provided by Supplier
which affect the availability of the Cloud Service; or
4. Caused by your use of Cloud Services other than expressly authorized by, and in accordance
with, the terms of the Agreement and the Order Form or Customer’s use of the Cloud
Services after we advised you to modify your use of the Service, if you did not modify your
use as advised.

> #### Exclusive Remedy

Reviewer 2: ```suggestion
#### Exclusive remedy
```

> Except as expressly set out in the Agreement, Customer acknowledges and agrees that

Reviewer 2: Not sure if all the capitalization is needed here

Author: That's a copy from a SAG document ... I also don't know but I reduced it to only the terms that are actually introduced in the text.

Supplier’s sole obligation and Customer’s exclusive remedy for Supplier’s failure to meet the
Service Availability requirements are set forth in this Service Credit Commitment.

### Support and maintenance

#### **Support**

* **Customer support:** Support is provided in accordance with the customer’s selected support plan (Starter, Standard, or Enterprise Active Support), as detailed in a separate support agreement.
* **Pre-production environments:** For pre-production environments, Starter-level support is generally provided, with support tickets handled at standard priority.

#### **Maintenance**

* **Ongoing maintenance and upgrades:** The {{< product-c8y-iot >}} platform is continuously maintained and upgraded to ensure optimal performance and security.

> * **Seamless upgrades:** This maintenance process is designed to be seamless and generally invisible to customers. The timing and content of upgrades are at the discretion of Cumulocity.

Reviewer 2: ```suggestion
* **Seamless upgrades:** This maintenance process is designed to be seamless and generally invisible to customers. The timing and content of upgrades are at the discretion of {{< company-c8y >}}.
```

> * **Upgrade information:** Details about scheduled upgrade times are available on the platform’s status pages as outlined above, while information about the specific changes included in each upgrade can be found in the [change logs](/change-logs/) within the customer documentation.

Reviewer 2: ```suggestion
* **Upgrade information:** Details about scheduled upgrade times are available on the platform’s status pages as outlined above, while information about the specific changes included in each upgrade can be found in the [change logs](/change-logs/) within the user documentation.
```

* **Regulated environments:** For customers operating in regulated environments, an optional annual maintenance schedule is available to meet specific compliance requirements.

### Acceptance

> By using the Services provided by Cumulocity, the Customer agrees to adhere to the terms outlined in this SLA.

Reviewer 2: ```suggestion
By using the Services provided by {{< company-c8y >}}, the Customer agrees to adhere to the terms outlined in this SLA.
```
